apps/examples/wc-react17/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "wc-react17",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-scripts": "5.0.1"
+  },
+  "scripts": {
+    "dev": "react-scripts start",
+    "build": "react-scripts build"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app"
+    ]
```

Comment:
I think the other example apps are using the JET eslint package

Reply:
Good spot!